public/locales/en/paymentCancelled.json

```diff
@@ -0,0 +1,4 @@
+{
+  "text": "The payment could not be made and registration for the event has been cancelled.",
```

Comment:
I'll leave it up to you to if you want to do this change: I could consider "The payment was not made and registration for the event has been cancelled." as an alternative to this phrase since it's a known user action causing the cancellation (either user goes back from bank without paying by declining the payment there or they go back from checkout to our service).

Reply:
Changed as suggested.